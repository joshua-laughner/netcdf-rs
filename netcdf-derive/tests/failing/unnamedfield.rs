#[repr(C)]
#[derive(netcdf_derive::NcType)]
struct UnnamedField(u8);

fn main() {}
