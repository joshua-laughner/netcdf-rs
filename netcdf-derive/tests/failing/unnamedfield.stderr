error: Can not derive NcType for struct with unnamed field
 --> tests/failing/unnamedfield.rs:3:20
  |
3 | struct UnnamedField(u8);
  |                    ^^^^
  |
  = note: #[derive(NcType)]
